.circleci/config.yml

```diff
@@ -2,11 +2,36 @@ version: 2
 jobs:
   make-eksctl-image:
     # use machine build, as otherwise we cannot use make and have to duplicate makefile
-    machine: true
+    machine:
+      image: ubuntu-1604:201903-01
```

Comment:
Isn't there an image that include Go toolchain?

Reply:
This was done for other reasons (see https://github.com/weaveworks/eksctl/pull/944/commits/51cfdfc509cf714eef400df9f27f4ec6b2102802 ).

Regardless, all the images contain old versions of Go (i.e. <1.11) which don't support go modules.

Reply:
I see, never mind :)